packages/create-audius-app/examples/react/index.html

```diff
@@ -5,6 +5,10 @@
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite + React + TS</title>
+    <script type="module">
```

Comment:
interesting, was this found from the tests?

Reply:
Yup! Specifically found by running the tests against a prod build. Because vite dev and prod is different